app/views/school/welcome_wizard/what_happens_next.html.erb

```diff
@@ -0,0 +1,25 @@
+<%- title = t('page_titles.school_user_welcome_wizard.what_happens_next.title') %>
+<%- content_for :title, title %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_for @wizard, url: school_welcome_wizard_path do |f| %>
+      <%= f.hidden_field :step %>
+      <h1 class="govuk-heading-xl"><%= title %></h1>
+
+      <p class="govuk-body">We’ll let your <%= @school.responsible_body.humanized_type %> know that you’re ready to place orders.</p>
```

Comment:
We'll need to omit this for:
- second time users
- single-academy trust users

Maybe just drop this line altogether for now.

Reply:
I've dropped this line.